app/models/tag.rb

```diff
@@ -1034,6 +1031,10 @@ def merger_string=(tag_string)
     end
   end
 
+  def unwrangled?
```

Comment:
Your comment on the difference between unwrangled and unwrangleable at https://github.com/otwcode/otwarchive/pull/4646#issuecomment-1840664875 is really good, you should include that on top of this method as well.